<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesView</class>
 <widget class="QWidget" name="PropertiesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>488</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Property Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollAreaProperties">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>304</width>
        <height>468</height>
       </rect>
      </property>
      <widget class="QWidget" name="propertiesObject" native="true">
       <property name="geometry">
        <rect>
         <x>-8</x>
         <y>-7</y>
         <width>311</width>
         <height>41</height>
        </rect>
       </property>
       <widget class="QPushButton" name="pushButtonAttachComponent">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string> Component</string>
        </property>
        <property name="icon">
         <iconset resource="../gameplay-editor.qrc">
          <normaloff>:/res/images/attach.png</normaloff>:/res/images/attach.png</iconset>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxEnabled">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>13</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditName">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="propertyEditors" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>301</width>
         <height>261</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="PropertiesTransform" name="propertiesTransform" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>301</width>
         <height>160</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAttachCamera">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-camera.png</normaloff>:/res/images/scene-camera.png</iconset>
   </property>
   <property name="text">
    <string>Attach Camera</string>
   </property>
  </action>
  <action name="actionAttachLight">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/scene-light-point.png</normaloff>:/res/images/scene-light-point.png</iconset>
   </property>
   <property name="text">
    <string>Attach Light</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PropertiesTransform</class>
   <extends>QWidget</extends>
   <header>PropertiesTransform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gameplay-editor.qrc"/>
 </resources>
 <connections/>
</ui>
